tests/framework/installer/installer.go

```diff
@@ -69,7 +69,12 @@ func SkipTestSuite(name string) bool {
 }
 
 func SystemNamespace(namespace string) string {
```

Comment:
IIRC, when we disabling the admission controller so it was failing to create the namespace so then we decided to create it manually.

Reply:
And, I'll test these with the latest changes and I missed a few things so, I'll be adding it.

Reply:
Since the admission controller is disabled now, can we keep the different system namespace for the operator? I'd like to keep a different namespace in the tests if at all possible.